external/wpa_supplicant/Kconfig

```diff
@@ -73,9 +73,9 @@ config DRIVER_T20
 	default y
 	depends on SCSC_WLAN
 
-config DISABLE_EAP_FOR_SUPPLICANT
-	bool "Disable EAP for WPA supplicant"
-	default y
+config ENABLE_EAP_FOR_SUPPLICANT
+	bool
+	default n
```

Comment:
Many users (developers) do NOT see source codes. To control Kconfig, they will see only kconfig file or menuconfig. Why don't you add help sentences to describe this like source codes?